.vscode/settings.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Check comments.

Reply:
I’m not the author of all that you listed here and you did not highlight or comment on any of my contributions.

Reply:
Cool cool. Can you check the issues in comment:
1. https://github.com/BanklessDAO/bounty-board/pull/246#issuecomment-1240763488
2. https://github.com/BanklessDAO/bounty-board/pull/246#issuecomment-1240767342
3. https://github.com/BanklessDAO/bounty-board/pull/246#issuecomment-1240780398

Reply:
I'll take a look at them, but probably not until Sunday

Reply:
Fixed the button spacing on claim

Reply:
Fixed:

- Button spacing issue
- Light mode issue
- Took out superfluous help boxes on bounty form
- Added a "Bounty List" nav button so you can get back after creating a bounty
- Fixed marking as paid to use new data structures that were added to the bot side
- Added Paid Status chicklet to the bounty summary strip
- Moved paid function behind the API, added an endpoint test (Eventually I think more of the higher level business logic also needs to go behind the API, but for later)